src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ContextQuery/CSharpSyntaxContext.cs

```diff
@@ -263,6 +265,7 @@ private static CSharpSyntaxContext CreateContextWorker(Document document, Semant
                 isEnumTypeMemberAccessContext: syntaxTree.IsEnumTypeMemberAccessContext(semanticModel, position, cancellationToken),
                 isFixedVariableDeclarationContext: syntaxTree.IsFixedVariableDeclarationContext(position, leftToken),
                 isFunctionPointerTypeArgumentContext: syntaxTree.IsFunctionPointerTypeArgumentContext(position, leftToken, cancellationToken),
+                isGenericConstraintContext: targetToken.Parent.IsKind(SyntaxKind.TypeParameterConstraintClause),
```

Comment:
this is not quite accurate right?  i'm presuming you only want to be in this context when after the `:` or `,` in a constraint clause.  but this check will report true when directly after the `where` token.  so if something *was* checking for this (thinking it was in the constraint portion, it would show up here `where $$`.

Reply:
Completions directly after the `where` token are implemented on recomendation service layer because syntaxes of generic constraints are different in C# and VB (I guess, didn't check that 🙂), so this thing doesn't affect them:
![devenv_NJTUt6NqXQ](https://user-images.githubusercontent.com/70431552/169151487-23c94ad3-1d2a-4576-b0e0-b6229506197c.png)

Reply:
i'm saying this can affect something in the future.  e.g. by only checking the parent, and not the token you're on, you're specifying three loctions as constraint locations, not the expected 2.

Reply:
Manually verified, that now `IsGenericConstraintContext` is `false` right after `where` keyword